hw/ip/usbdev/doc/_index.md

```diff
@@ -144,6 +144,28 @@ To control the pullup on the D- line, this USB device features `dn_pullup_o` sig
 Of the two pullup signals `dp_pullup_o` and `dn_pullup_o`, only one can be enabled at any time.
 As this is a Full-Speed device, `dp_pullup_o`, i.e., the pullup on D+ is used by default (`pinflip` equals zero).
 
+### FPGA board implementation with PMOD
+
+The interface was developed using the Digilent Nexys Video board with a PMOD card attached.
+A PMOD interface with direct connection to the SoC should be used (some PMOD interfaces include 100R series resistors which break the signal requirements for USB).
+The PMOD card includes two USB micro-B connectors and allows two USB interfaces to be used.
+The D+ and D- signals have 22R series resistors (in line with the USB spec) and there is a 1.5k pullup on D+ to the pullup enable signal.
+There is a resistive divider to set the sense pin at half of the VBUS voltage which enables detection on the FPGA without overvoltage on the pin.
+
+![PMOD Schematic](dualpmod-sch.svg)
+
+The PMOD pcb is [available from OSH Park](https://oshpark.com/shared_projects/xMKhTIHn).
+
+The PMOD design files for Kicad version 5 are in the `usbdev/pmod` directory. 
```

Comment:
`usbdev/pmod` -> [`usbdev/pmod`](https://github.com/lowRISC/opentitan/tree/master/hw/ip/usbdev/pmod)

With the embedded link (sorry, don't know how to un-render this).

Reply:
Kicad -> KiCad?

Reply:
done